frontend/src/components/QueueTable.js

```diff
@@ -58,9 +65,21 @@ function QueueTable({ queueData: { name, rows } }) {
   return (
```

Comment:
could be cool to add some logic for if the queues are still loading and what to display.  I started doing something akin to this on the Participant List component on my branch here: https://github.com/CodeForPhilly/prevention-point/blob/Search_Participant_New/frontend/src/components/ParticipantsList.js.  Take a look at the useEffect hook for what I did.  Right now on your branch here, it will display `no results` or something like that in the interim as its getting the queues and then get display the queue data once its fetched.  But thats not exactly correct.  It maybe cool to have a simple loading state to let the user know that the data is on the way!  Let me know if you have questions about how I implemented it.  Its also possible that later on we should add a loading state component to display a loading icon or something, but for now we can just use some basic loading text

Reply:
I'll see if I can get me head around it

Reply:
I'll see if I can get me head around it

Reply:
The way the mobx documentation does this is to create an observable in the store (naturally) that reflects the loading status of the queue.

https://mobx.js.org/best/actions.html#flows

Reply:
I took a shot at doing this using the pattern from the mobx documentation, keeping track of the loading state.  I led to an infinite loop of db fetches and re-rendering, so some rethinking is in order to figure out the best way to accomplish this

Reply:
I think we can leave this for another ticket.  I think if you are thinking of it in a global way as opposed to a component based loader like I have (using useState), then it should be a separate ticket that we can apply across the components.  I will make a ticket